server/api_server/tools_routes.py

```diff
@@ -0,0 +1,686 @@
+import subprocess
+import re
+from datetime import datetime, timedelta
+from flask import Blueprint, request, jsonify
+import sqlite3
+from helper import get_setting_value
+from database import get_temp_db_connection
+
+tools_bp = Blueprint('tools', __name__)
+
+
+def check_auth():
+    """Check API_TOKEN authorization."""
+    token = request.headers.get("Authorization")
+    expected_token = f"Bearer {get_setting_value('API_TOKEN')}"
+    return token == expected_token
+
+
+@tools_bp.route('/trigger_scan', methods=['POST'])
```

Comment:
ideally all routes should be in the api_server_start file so that it's easy to see available routes in one place.